### PR TITLE
vet and gofmt

### PR DESCRIPTION
**Pull request by alicebob**
Fix some `go vet` warnings (and run `gofmt` while we're at it).

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -1022,8 +1022,6 @@ func (clnt *Client) QueryAggregate(policy *QueryPolicy, statement *Statement, pa
 			luaArgs...,
 		); err != nil {
 			panic(err)
```
Comment (khaf) [inline client.go]: It is the panic which should be removed.

File: client_object_test.go [hunk 2]
```diff
@@ -536,7 +536,6 @@ var _ = Describe("Aerospike", func() {
 
 				for e := range rs.Errors {
 					panic(e.Error())
```
Comment (khaf) [inline client_object_test.go]: Same here!

File: client_object_test.go [hunk 3]
```diff
@@ -578,7 +577,6 @@ var _ = Describe("Aerospike", func() {
...
```
Comment (khaf) [inline client_object_test.go]: And here!

File: pkg/bcrypt/cipher.go [hunk 4]
```diff
@@ -272,10 +272,9 @@ var bf_crypt_ciphertext = [6]uint{
 	0x64657253, 0x63727944, 0x6f756274,
 }
 
-
 type cipher struct {
-	P [18]uint
```
Comment (khaf) [inline pkg/bcrypt/cipher.go]: This package is external; I'd rather not modify it.

**Comment by khaf** [comment]
Thanks for the PR.

Looks good. If you remove the panics and return the original code, I'll merge it in.

**Comment by alicebob** [comment]
Applied your comments. It's a push -f to clean up the panic() removals.

**Comment by khaf** [comment]
Merged. Thanks.

**Pull request merged**